readthedocs/core/utils/filesystem.py

```diff
@@ -127,21 +97,11 @@ def safe_copytree(from_dir, to_dir, *args, **kwargs):
 
 def safe_rmtree(path, *args, **kwargs):
     """
-    Wrapper around shutil.rmtree to check for symlinks.
+    Wrapper around shutil.rmtree for security reasons.
 
     shutil.rmtree doens't follow symlinks by default,
-    this function just logs in case users are trying to use symlinks.
     https://docs.python.org/3/library/shutil.html#shutil.rmtree
 
     The extra *args and **kwargs will be passed to the rmtree() function.
     """
-    path = Path(path)
-    if path.is_symlink():
-        log.info(
-            "Not deleting directory because it's a symlink.",
-            path=str(path),
-            resolved_path=path.resolve(),
-        )
-        return None
-    _assert_path_is_inside_docroot(path)
```

Comment:
rmtree fails hard (OSError) when trying to delete a dir that points to a symlink, but `_assert_path_is_inside_docroot` is still useful as mentioned in my other comment.